### DISCUSSION TITLE
NightDriverServer repository released

### DISCUSSION THREAD

**Discussion by rbergen** [PlummersSoftwareLLC/NightDriverStrip#186] [Announcements]
Hello everybody,

This is just a quick message to let you know that the [NightDriverServer](https://github.com/PlummersSoftwareLLC/NightDriverServer) repository has now been made public. As Dave puts it 'the purpose of the NightDriverServer project is to serve as a "demo" of how to send color data over WiFi to a NightDriverStrip instance on an ESP32'. If this is something you're interested in trying or investigating, please do go and have a look!

The repository's [README](https://github.com/PlummersSoftwareLLC/NightDriverServer/blob/master/README.md) provides more information about the NightDriverStrip network packet format, including timing considerations for scenarios in which synchronization across multiple devices is required.

Cheers,
Rutger

**Comment by xvlvx0** [comment]
Hi Rutger, Dave,

I have been looking around in the 'NightDriverServer' repo and noticed the Docker compose file. Which made me search for it....and it is still hosted on Docker.

I got it running and it service the wwwroot folder and its contents, but....what now! The documentation isn't that extensive on what it can do. I noticed that the container has this project compiled ready to go in the '/docker' folder.
Can you write up a tiny bit more context on how to use it? Or please point me to the correct documentation pages.

BR,
xvlvx

**Reply by rbergen** [reply]
@xvlvx0 I apologize for the "GitHub bookkeeper" response, but could you please make this question a Q&A discussion in the NightDriverServer repo itself? It'll make it much easier to track what relates to what, and link it to any future changes in code or documentation.

**Reply by xvlvx0** [reply]
Consider it done!
